src/modules/auth/AccessControlScreen.js

```diff
@@ -54,51 +55,84 @@ function AccessControlCard(props) {
 	if (App.Services.TenantService) {
 		currentTenant = App.Services.TenantService.get_current_tenant();
 	}
+	const advmode = useSelector(state => state.advmode?.enabled);
+	const theme = useSelector(state => state.theme);
 
-	return(
-		<Card className="shadow animated fadeIn">
-			<CardHeader className="text-center border-bottom card-header-login">
-				<div className="card-header-title">
-					<CardTitle className="text-primary mb-0">
-						{t('AccessControlScreen|Access control')}
-					</CardTitle>
-					<CardSubtitle tag="p">
-						{t('AccessControlScreen|See your details')}
-					</CardSubtitle>
-				</div>
-			</CardHeader>
-
-			<CardBody>
-				{ userinfo ? (
-					<>
-						{App.Services.TenantService &&
-							<React.Fragment>
-								<Row>
-									<Col>
-										{t('AccessControlScreen|Tenant')}
-									</Col>
-									<Col>
-										<p style={{marginBottom: "5px"}}>{currentTenant}</p>
-									</Col>
-								</Row>
-								<hr/>
-							</React.Fragment>
-						}
-						<Row>
-							<Col>
-								{t('AccessControlScreen|Resources')}
-							</Col>
-							<ItemToRender userinfo={userinfo} resources={resources} item='resources' currentTenant={currentTenant} />
-						</Row>
-					</>
-					) : (
-					<div className="text-center">
-						{t("AccessControlScreen|The user information is invalid, you session is likely expired.")}
-					</div>
-				)}
-			</CardBody>
-		</Card>
-		)
+	return (
+		<Container>
+			<Row>
+				<Col>
+					<Card className="shadow animated fadeIn">
+						<CardHeader className="text-center border-bottom card-header-login">
+							<div className="card-header-title">
+								<CardTitle className="text-primary mb-0">
+									{t('AccessControlScreen|Access control')}
+								</CardTitle>
+								<CardSubtitle tag="p">
+									{t('AccessControlScreen|See your details')}
+								</CardSubtitle>
+							</div>
+						</CardHeader>
+
+						<CardBody>
+							{ userinfo ? (
+								<>
+									{App.Services.TenantService &&
+										<React.Fragment>
+											<Row>
+												<Col>
+													{t('AccessControlScreen|Tenant')}
+												</Col>
+												<Col>
+													<p style={{marginBottom: "5px"}}>{currentTenant}</p>
+												</Col>
+											</Row>
+											<hr/>
+										</React.Fragment>
+									}
+									<Row>
+										<Col>
+											{t('AccessControlScreen|Resources')}
+										</Col>
+										<ItemToRender userinfo={userinfo} resources={resources} item='resources' currentTenant={currentTenant} />
+									</Row>
+								</>
+							) : (
+								<div className="text-center">
+									{t("AccessControlScreen|The user information is invalid, you session is likely expired.")}
+								</div>
+							)}
+						</CardBody>
+					</Card>
+				</Col>
+			</Row>
+
+			<Row className="justify-content-md-center mt-4">
```

Comment:
@Aringoaway I think it would be better to wrap the whole `Row` to `advmode` than just the content of the row. Also I am not sure about the justification of the content of the Row (pls take a look how other advanced mode cards look like in other screens)